RecoTracker/TrackProducer/interface/TrackProducerAlgorithm.h

```diff
@@ -136,6 +138,7 @@ class TrackProducerAlgorithm {
   reco::TrackBase::TrackAlgorithm algo_;
   bool reMatchSplitHits_;
   bool geometricInnerState_;
+  bool usePropagatorForPCA_;
```

Comment:
this may actually explain what I see: after rerunning with igprof I get different results compared to without it